wiki/Article_styling_criteria/Writing/th.md

```diff
@@ -0,0 +1,336 @@
+# การเขียน
+
+*สำหรับมาตรฐาน การจัดรูปแบบ, ดูที่: [เกณฑ์รูปแบบบทความ/การจัดรูปแบบ](../Formatting)*
+
+*ข้อสังเกต: บทความนี้ใช้ [RFC 2119](https://tools.ietf.org/html/rfc2119) เพื่ออธิบายระดับความต้องการ*
```

Comment:
this is also pointless (see my note on RFC somewhere below)

Reply:
uh english article included this so i put it in here. is it okay?

Reply:
https://tools.ietf.org/html/rfc2119 explains how certain key words should be interpreted. these keywords are: "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL". if your translation uses neither of these, it might be a good idea to drop this line, because it makes no sense.